### PR TITLE
Friendship v2

### PR DESCRIPTION
**Pull request by Alexoid1**
Table friendship with user_id, friend_id and confirmed columns that gives a boleean value if there is friendship or not

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ AllCops:
   DisplayCopNames: true
 
 Layout/LineLength:
-  Max: 120
+  Max: 180
```
Comment (lekegitrepo) [inline .rubocop.yml:13]: - **_Do not modify any of the linter files. Please, restore the original value that was modified_**

**Review by lekegitrepo — CHANGES_REQUESTED** [review]
## Status: Request changes :recycle:
### Hi @Alexoid1 :clap: :clap:
#### Good job so far. You need to implement some missing requirements. Check the requirements :point_down:

- **_Enable linter for this repo_**

- **_Follow the Git flow as mentioned in the project requirement_**

- **_when friendship is confirmed, an additional row with reversed user-friend pair should be created_**

- **_PostsController#index should displays posts in a timeline (their own posts + posts from their friends) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/final_project/timeline.png) from [Odin Project](https://www.theodinproject.com/courses/ruby-on-rails/lessons/final-project)). Right now this is showing all the users(friends or not) post_**

### Optional:
  **_You can read about `n+1` and make sure there are no `n + 1` queries in index views [Recommended article](https://medium.com/@bretdoucette/n-1-queries-and-how-to-avoid-them-a12f02345be5)._**


#### _Reviewed by Temitayo Adeleke_

File: .rubocop.yml (hunk 1 above)
Comment (Alexoid1) [reply .rubocop.yml:13]: friend = Friendship.where(user_id: @friendship.friend_id)
    friend.find_by(friend_id: @friendship.user_id).update(confirmed: true)
that is the reversed user_friend a crerate another column in the same time that confirmed the friendship check the code the method is there do the same the only thing is that dont have that name of reversed friendship

Comment (Alexoid1) [reply .rubocop.yml:13]: also in the model i dont call it reverse_frienships i just call it friends

has_many :friends, foreign_key: 'friend_id', class_name: 'Friendship'

i think  to call it reverse_friendship is no  mandatory or I'm wrong???

**Pull request closed without merging**